pkg/k8s/client/client.go

```diff
@@ -35,69 +36,50 @@ const (
 )
 
 var (
-	client         *kubernetes.Clientset
-	config         *rest.Config
-	namespace      string
-	currentContext string
-	localClusters  = []string{"127.", "172.", "192.", "169.", model.Localhost, "::1", "fe80::", "fc00::"}
+	client           *kubernetes.Clientset
+	config           *rest.Config
+	clientConfig     clientcmd.ClientConfig
+	currentNamespace string
+	currentContext   string
+	localClusters    = []string{"127.", "172.", "192.", "169.", model.Localhost, "::1", "fe80::", "fc00::"}
 )
 
 //GetLocal returns a kubernetes client with the local configuration. It will detect if KUBECONFIG is defined.
-func GetLocal(k8sContext string) (*kubernetes.Clientset, *rest.Config, string, error) {
+func GetLocal(currentContext string) (*kubernetes.Clientset, *rest.Config, error) {
```

Comment:
I'd also change this to only return the k8s client, and have a separate function to get the REST config, since most of the calls only need the first one.